spec/orders.json

```diff
@@ -1,33 +1,45 @@
 {
-  "host": "api.shopbonsai.ca/orders",
-  "schemes": ["https"],
-  "basePath": "/",
+  "host": "api.shopbonsai.ca",
+  "schemes": [
+    "https"
+  ],
+  "basePath": "/orders",
   "swagger": "2.0",
   "info": {
     "title": "Orders API",
-    "description": "APIs used for interacting with orders.",
-    "version": "2.1.2",
-    "contact": { "email": "developers@shopbonsai.ca" }
+    "description": "\n  APIs used for interacting with orders.\n\n  There are a few types of integration with the Bonsai Order API depending on who owns the payments.\n\n  ## Bonsai owns the payment\n\n  ### Native Checkout experience\n\n  If during the checkout process, the customer is redirected to Bonsai's checkout page to complete payment and order processing.\n\n  ### API integrated checkout experience\n\n  If you are hosting your own checkout experience and leveraging our endpoints to collect payment and complete order processing.\n  Please contact our team to provide you with all you need to get started.\n\n  ## Account owns the payment\n\n  If you are owning payment. Orders will need to be submitted with the payment method `external payment`.\n  Once the order is submitted, our team will handle fulfillment and Bonsai will invoice you for the order amount based on our agreement.\n  ",
+    "version": "3.8.13",
+    "contact": {
+      "email": "developers@shopbonsai.ca"
+    }
   },
   "paths": {
     "/checkout": {
       "post": {
-        "tags": ["Checkouts"],
-        "produces": ["application/json"],
+        "tags": [
```

Comment:
90% of these are because when I use format json in VScode, it seems to split this differently. We might want to have some json linter added so it's not dependant on who formats it locally